Rakefile

```diff
@@ -36,4 +36,22 @@ Rake::TestTask.new do |t|
   t.verbose = true
 end
 
-task default: [:test, :rubocop]
+task default: [:test, 'rubocop:auto_correct']
```

Comment:
why's that?

the auto correction that it does is just easy style things (you still need to do the more complicated ones yourself)